demo/package.json

```diff
@@ -47,7 +47,6 @@
     "babel-preset-es2015": "^6.3.13",
     "babel-preset-react": "^6.3.13",
     "babel-preset-stage-2": "^6.3.13",
-    "brfs-babel": "^1.0.0",
```

Comment:
theoretically no we don't need it, but removing transform-loader from devDep completely caused tests to fails. (we are still using fs.readFileSync in the tests, I'd assume it is handled by node.js so we should not need transform-loader but it failed in my first trial.

Since transform-loader is in devDep only, I'd prefer to land this PR first to release the rc.2 (a blocker).